packages/esm-reports-app/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@openmrs/esm-reports-app",
+  "version": "4.0.1",
+  "license": "MPL-2.0",
+  "description": "Reports Admin page for OpenMRS",
+  "browser": "dist/openmrs-esm-reports-app.js",
+  "main": "src/index.ts",
+  "source": true,
+  "scripts": {
+    "start": "openmrs develop",
+    "serve": "webpack serve --mode=development",
+    "build": "webpack --mode production",
+    "analyze": "webpack --mode=production --env.analyze=true",
+    "lint": "eslint src --ext js,jsx,ts,tsx",
+    "typescript": "tsc",
+    "test": "cross-env TZ=UTC jest --config jest.config.js --verbose false --passWithNoTests --color",
+    "test:watch": "cross-env TZ=UTC jest --watch --config jest.config.js --color",
+    "extract-translations": "i18next 'src/**/*.component.tsx' --config ../../tools/i18next-parser.config.js"
+  },
+  "browserslist": [
+    "extends browserslist-config-openmrs"
+  ],
+  "keywords": [
+    "openmrs",
+    "microfrontends",
+    "reports"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openmrs/openmrs-esm-admin-tools.git"
+  },
+  "homepage": "https://github.com/openmrs/openmrs-esm-admin-tools#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/openmrs/openmrs-esm-admin-tools/issues"
+  },
+  "dependencies": {
+    "@carbon/react": "^1.33.1",
+    "@datasert/cronjs-matcher": "^1.2.0",
+    "@datasert/cronjs-parser": "^1.2.0",
+    "cronstrue": "^2.41.0",
+    "dayjs": "^1.8.36",
+    "lodash-es": "^4.17.21",
+    "react-image-annotate": "^1.8.0"
+  },
+  "peerDependencies": {
+    "@openmrs/esm-framework": "*",
+    "dayjs": "1.x",
+    "react": "18.x",
+    "react-i18next": "11.x",
+    "react-router-dom": "6.x",
+    "rxjs": "6.x"
+  },
+  "packageManager": "yarn@3.6.1"
```

Comment:
We can upgrade to Yarn v4 by running `corepack enable && yarn set version stable && yarn install`

Reply:
fixed

Reply:
Could you commit the folders in your local `.yarn` directory?

Reply:
Could you commit the folders in your local `.yarn` directory?

Reply:
done